### ISSUE TITLE
uBlock now supresses user login for nationalpost.com under firefox

### ISSUE THREAD

**Issue by MrMoxy** [uBlockOrigin/uAssets#24654]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://nationalpost.com/
```

### Description

As of this week, uBlock blocks the user sign in prompt on the masthead and any attempt to log in yields a time out. This is a NWE issue and disappears when uBlock is disabled. It works fine under Chrome, so is likely specific to Firefox (latest Update).

### Other extensions used

Privacy badger. However, uBlock is the issue because disabling it fixes the problem

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml


```
</details>

**Comment by mapx-** [comment]
add

`@@||cdn.sophi.io/assets/$script,domain=nationalpost.com`

@ryanbr

**Comment by MrMoxy** [comment]
Part way there. That addition caused the login link to reappear on the site masthead. However clicking on it still results in a infinite wait and timeout with the site never returning.

Thanks.

**Comment by stephenhawk8054** [comment]
Don't remove the troubleshooting information.

The website has issue themselves. Even when I don't use any extensions, the screen is still stuck there. Sometimes it works, sometimes it's stuck. It's unstable.

**Comment by gmow1** [comment]
Same issue, being a newbie where do I add 
@@||cdn.sophi.io/assets/$script,domain=nationalpost.com

as suggested by @mapx-

**Comment by arcanistzed** [comment]
### Browser name and version

Google Chrome: 148.0.7778.178 (Official Build) (64-bit) 

### Other extensions used

none

### Country

Canada

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

<table>
<tr>
 <td>
<img width="1030" height="1112" alt="Image" src="https://github.com/user-attachments/assets/6788949b-616c-4b35-94f5-7a36f77845a4" />
 <td>
<img width="1030" height="1112" alt="Image" src="https://github.com/user-attachments/assets/2d21bcba-0ea9-40df-b856-0e19f7563b05" />
<tr>
 <td>
Filtering mode: optimal
 <td>
Filtering mode: no filtering
</table>

</details>


### Configuration

```yaml
name: uBlock Origin Lite
version: 2026.516.1652
browser: Chrome 148 (linux)
filtering:
 site: optimal
 default: optimal
permission: all
badge: 14
rules: 22519
rulesets:
 easylist
 easyprivacy
 fra-0
 pgl
 ublock-badware
 ublock-filters
 urlhaus-full
scripting:
 ublock-filters.main
 ublock-filters.isolated
 easyprivacy.main
 easyprivacy.isolated
 ublock-badware.main
 fra-0.main
 fra-0.isolated
 prevent-popup
 toolbar-icon
 css-specific
 css-procedural
console:
 [0529.2234]insertCSS/Error: Frame with ID 851 was removed.
 [0529.2234]executeScript/Error: Frame with ID 851 was removed.
 [0530.0025]insertCSS/Error: No frame with id 2393 in tab with id 1695365138
 [0530.0025]executeScript/Error: No frame with id 2393 in tab with id 1695365138
```

**Comment by stephenhawk8054** [comment]
It will be fixed in version `2026.529.1448`.

**Comment by walkertonyr-boop** [comment]
This problem seems to have reappeared on the current Google Chrome.

**Comment by walkertonyr-boop** [comment]
I'm new to these forums and not sure how this works. I entered a problem with National Post (#34473) a week ago. It has been closed and marked as a duplicate of this problem, but this problem was specific to Firefox and is marked closed even though it has resurfaced in Chrome.  Is anyone looking at this?

**Comment by stephenhawk8054** [comment]
We said in the prerequisites that you ticked on

> - [x] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.). **Comment in the old issue threads even when they are closed or even if you have a different problem.**

Comment in the old issue threads even when they are closed or even if you have a different problem.
--

even when they are closed or even if you have a different problem.
--

**Comment by walkertonyr-boop** [comment]
Yes, I understand that and I did comment on the old issue (this one). Since the problem has re-appeared, and in a different context (Chrome, not Firefox), it needs to be looked at, but this issue is still marked as closed. Is it being worked on?

**Comment by gothic-bum** [comment]
@walkertonyr-boop 
  
Post the troubleshooting information as described [here](https://github.com/uBlockOrigin/uAssets#how-to-correctly-report-an-issue).

**Comment by walkertonyr-boop** [comment]
Copied from #34473 

URL address of the web page
https://nationalpost.com/

Category
breakage

Description
When displaying this page, uBO does a great job of filtering out unwanted images and videos. Unfortunately it also suppresses the auto-sign-in feature and the ability to sign in manually. Clicking the 'Sign in' button should go to a sign-in page but uBO suppresses the page.

Browser name and version
Google Chrome, Version 153.0.8010.36 (Official Build) (64-bit)

Other extensions used
Acrobat, One Note, Web of Trust, Google Dictionary, Google Scholar, Google Translate.
Issue persists will all disabled.

Country
Canada

Screenshot(s)

<details>

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/12e74bdd-81ce-49a7-a158-480d183dce6d" />

</details>

Configuration
Details

<details>

```yml
name: uBlock Origin Lite
version: 2026.907.2003
browser: Chrome 153 (win)
filtering:
 site: optimal
 default: optimal
permission: all
badge: 18
rules: 18784
rulesets:
 easylist
 easyprivacy
 pgl
 ublock-badware
 ublock-filters
 urlhaus-full
scripting:
 ublock-filters.main
 ublock-filters.isolated
 easyprivacy.main
 easyprivacy.isolated
 ublock-badware.main
 ublock-badware.isolated
 prevent-popup
 css-specific
```

</details>

**Comment by walkertonyr-boop** [comment]
When you say "Comment in the old issue threads even when they are closed or even if you have a different problem.", maybe you should be more specific about what content you want in the comment.

**Comment by gothic-bum** [comment]
@walkertonyr-boop 

1\. Update to uBO Lite 2026.920.1710

2\. Restart your browser

3\. Open a new page/tab and test again
